tests/testthat/test_overall.R

```diff
@@ -326,3 +326,29 @@ test_that("mod_marg2 input is checked", {
                            at = list(age = 100)))
 })
 
+test_that("Setting base level works", {
+
+  # extrapolated values are troubling
+  mm <- glm(y ~ as.factor(treatment) + age, margex, family = 'gaussian')
+
+  # Setting base_rn without type = 'effects' does nothing (warning)
+  expect_warning(mod_marg2(mod = mm, var_interest = 'treatment',
+                           at = list(age = 50), base_rn = 2))
+
+  # Actual check
+  z1 <- mod_marg2(mod = mm, var_interest = 'treatment',
+                  at = list(age = 50), base_rn = 1,
+                  type = 'effects')[[1]]
+  z2 <- mod_marg2(mod = mm, var_interest = 'treatment',
```

Comment:
@anniejw6 added stata output and raw numbers to the tests. The z2 model is already checked in the tests below; adding a check for z2 would just mean flipping those and would pass by definition. Do you have something else in mind?